### PR TITLE
Fix density bugs for TreeView, ComboBox and NavigationView

### PR DESCRIPTION
**Pull request by licanhua**
We extracted parameters for density, but we didn't actually downlevel TreeView and NavigationView RS5 density change to RS1. This is the pre-step for Compact.
Changes includes:
1. Remove AcrylicBrush for NavigationViewExpandedPaneBackground for RS4 and downlevel RS5 value to RS1. The background color is changed.
2. ToggleButton/CompactPaneLength/BackButton width from 48 to 40 for RS4 and below release.
3. Height of TreeViewItem/NavigationViewItem OnLeft navigationview to 32 and 40.
4. Combine MUX_NavigationViewItemPresenterStyleWhenOnLeftPane style on rs1 and rs2 into one.
5. I also make some changes to combobox style

### CODE REVIEW

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
     xmlns:primitives="using:Microsoft.UI.Xaml.Controls.Primitives"
+    xmlns:contract4Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,4)"
+    xmlns:contract4NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,4)"
     xmlns:media="using:Microsoft.UI.Xaml.Media">
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Dark">
             <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="SystemChromeMediumColor" />
-            <StaticResource x:Key="NavigationViewExpandedPaneBackground" ResourceKey="SystemChromeMediumHighColor" />
+            <StaticResource x:Key="NavigationViewExpandedPaneBackground" ResourceKey="SystemChromeMediumColor" />
```
Comment (chrisglein) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml]: Your change description didn't explicitly mention any color changes. Is this expected? Just a result of the merge down and reveal stuff?

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 2]
```diff
@@ -579,7 +604,7 @@
                 <ControlTemplate TargetType="primitives:NavigationViewItemPresenter">
                     <Grid
                         x:Name="LayoutRoot"
-                        Height="40"
+                        MinHeight="{ThemeResource NavigationViewItemOnLeftMinHeight}"
```
Comment (chrisglein) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml]: You changed from Height to MinHeight. Expected?

File: dev/CommonStyles/ComboBox_themeresources.xaml [hunk 3]
```diff
@@ -356,8 +356,12 @@
     <x:Int32 x:Key="ComboBoxPopupMaxNumberOfItems">15</x:Int32>
     <x:Int32 x:Key="ComboBoxPopupMaxNumberOfItemsThatCanBeShownOnOneSide">7</x:Int32>
 
+    <Thickness x:Key="ComboBoxPadding">12,5,0,7</Thickness>
```
Comment (chrisglein) [inline dev/CommonStyles/ComboBox_themeresources.xaml]: Your PR description only talks about TreeView and NavigationView. So, ComboBox changes too?

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 1 above)
Comment (licanhua) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml]: yes. It's expected.

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 2 above)
Comment (licanhua) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml]: yes.

File: dev/CommonStyles/ComboBox_themeresources.xaml (hunk 3 above)
Comment (licanhua) [reply dev/CommonStyles/ComboBox_themeresources.xaml]: I renamed the title

**Pull request merged**